backends/gdx-backend-headless/pom.xml

```diff
@@ -18,6 +18,12 @@
       <artifactId>gdx</artifactId>
       <version>${project.version}</version>
     </dependency>
+      <dependency>
+          <groupId>com.google.android</groupId>
```

Comment:
Why does the headless backend need to depend on Android?

Reply:
You're right, I'll refactor them properly. Thanks.